development.adoc

```diff
@@ -18,7 +19,10 @@ running.
 
 [source,shell]
 ----
-go get -u github.com/christopherhein/aws-operator-codegen
+cd code-generation
+make install-bindata
+cd ..
+make install-aws-codegen
```

Comment:
can you update the Makefile at the root of the project so this is a single step run from the root?

ie. make install-aws-codegen -> installs-bindata and installs aws-codegen